src/components/Layout/Header/styles.ts

```diff
@@ -0,0 +1,64 @@
+import { makeStyles } from '@material-ui/styles'
+import { Theme } from '@material-ui/core/styles'
+
+const drawerWidth = 240
+
+export default makeStyles((theme: Theme) => ({
+  root: {
+    display: 'flex',
+    flexGrow: 1
+  },
+  menuButton: {
+    marginRight: theme.spacing(2)
+  },
+  appBar: {
+    transition: theme.transitions.create(['margin', 'width'], {
+      easing: theme.transitions.easing.sharp,
+      duration: theme.transitions.duration.leavingScreen
+    })
+  },
+  appBarShift: {
+    width: `calc(100% - ${drawerWidth}px)`,
+    marginLeft: drawerWidth,
+    transition: theme.transitions.create(['margin', 'width'], {
+      easing: theme.transitions.easing.easeOut,
+      duration: theme.transitions.duration.enteringScreen
+    })
+  },
+  hide: {
+    display: 'none'
+  },
+  drawer: {
+    width: drawerWidth,
+    flexShrink: 0
+  },
+  drawerPaper: {
+    width: drawerWidth
+  },
+  drawerHeader: {
+    display: 'flex',
+    alignItems: 'center',
+    padding: '0 8px',
+    ...theme.mixins.toolbar,
+    justifyContent: 'flex-end'
+  },
+  content: {
+    flexGrow: 1,
+    padding: theme.spacing(3),
```

Comment:
Basically what's happening here is that u are setting an 24px of padding. How does it work?
theme has a pre-defined value for what 1 unit of spacing is .. in general 1 unit of spacing is = 8px. So over there what u did was to set padding: 8 * 3 = 24px.

Another way to use it:

`padding: theme.spacing(1, 3)` = `padding: 8px 24px` 

U can also read more about spacings here https://material-ui.com/system/spacing/